res/controllers/Traktor-Kontrol-S4-MK2-hid-scripts.js

```diff
@@ -0,0 +1,1285 @@
+/****************************************************************/
+/*      Traktor Kontrol S4 MK2 HID controller script v1.00      */
+/*      Copyright (C) 2015, the Mixxx Team                      */
+/*      but feel free to tweak this to your heart's content!    */
+/*      For Mixxx version 2.0                                   */
+/****************************************************************/
+
+// TODO:
+// * Soft takeover for rate slider
+// * Loop size readout / selection
```

Comment:
IMO it would be good to implement this before merging. How hard do you think it would be to implement this?